3.10/graphs-smart-graphs-getting-started.md

```diff
@@ -1,10 +1,9 @@
 ---
 layout: default
-description: SmartGraphs enable you to manage graphs at scale.
-title: ArangoDB SmartGraphs - Getting Started
+description: >-
+  SmartGraphs enable you to manage graphs at scale
```

Comment:
Personally, I'd like to see a more detailed summary or technical aspects here

Reply:
will take care of this together with the other graph related tasks.